src/runtime_src/core/common/xclbin_parser.h

```diff
@@ -21,6 +21,8 @@
 #include "xclbin.h"
 #include <string>
 #include <vector>
+#include <stdexcept>
```

Comment:
Please sort the `#include`

Reply:
Hi @keryell ,

you should I re-submit the patch with alphabetically sorted "include" ?

Reply:
If you do not mind, you can contribute even more into code cleaning. \o/

Reply:
Ok, no problem. Should I invalidate this commit before? How?

Reply:
> Patch resubmitted with suggestions from @keryell

I am unsure about what you meanwith *patch* here.
With `git` you just add a new commit with the new content and you push on your published branch.

Reply:
A commit which contains a patch...